portal.go

```diff
@@ -668,12 +668,13 @@ func (portal *Portal) sendMainIntentMessage(content interface{}) (*mautrix.RespS
 	return portal.sendMessage(portal.MainIntent(), event.EventMessage, content, 0)
 }
 
+const doublePuppetKey = "fi.mau.double_puppet_source"
+const doublePuppetValue = "mautrix-imessage"
+
 func (portal *Portal) sendMessage(intent *appservice.IntentAPI, eventType event.Type, content interface{}, timestamp int64) (*mautrix.RespSendEvent, error) {
 	wrappedContent := event.Content{Parsed: content}
 	if timestamp != 0 && intent.IsCustomPuppet {
-		wrappedContent.Raw = map[string]interface{}{
-			"net.maunium.imessage.puppet": intent.IsCustomPuppet,
-		}
+		wrappedContent.Raw[doublePuppetKey] = doublePuppetValue
```

Comment:
`Raw` isn't initialized here, that's why it was assigning to the whole map. It should be fine to use a constant as the map key (`{ doublePuppetKey: intent.IsCustomPuppet }`)